src/main/java/io/cdap/plugin/google/common/GoogleDriveBaseConfig.java

```diff
@@ -126,6 +192,28 @@ private void validateDirectoryIdentifier(FailureCollector collector, GoogleDrive
     }
   }
 
+  protected boolean checkPropertyIsSet(FailureCollector collector, String propertyValue, String propertyName,
+                                       String propertyLabel) {
+    if (!containsMacro(propertyName)) {
+      if (Strings.isNullOrEmpty(propertyValue)) {
+        collector.addFailure(getIsSetValidationFailedMessage(propertyLabel),
+                             getValidationFailedCorrectiveAction(propertyLabel))
+          .withConfigProperty(propertyName);
+      } else {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  protected String getIsSetValidationFailedMessage(String propertyLabel) {
+    return String.format(IS_SET_FAILURE_MESSAGE_PATTERN, propertyLabel);
+  }
+
+  protected String getValidationFailedCorrectiveAction(String propertyLabel) {
+    return String.format(CHECK_CORRECTIVE_MESSAGE_PATTERN, propertyLabel);
+  }
```

Comment:
why do we need this one line method?

Reply:
I was going to use it in configs for source, removed, will add back in source PR